frontend/src/widgets/ResourcesDashboardOverview.js

```diff
@@ -14,6 +19,7 @@ import colors from '../colors';
 export function Field({
```

Comment:
I am confused why this is duplicated from DashboardOverview rather than reusing it, but it looks like it was already like that